packages/expo-modules-autolinking/src/ReactImportsPatcher.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+import glob from 'fast-glob';
+import fs from 'fs-extra';
```

Comment:
With `fs/promises`, `fs-extra` is unnecessary here.

Reply:
i'm hesitant to change this simply because there are other fs-extra code in expo-modules-autolinking. i'm just trying to align the style here. if it's better to rewrite fs-extra as `fs/promise` also in other code of expo-modules-autolinking. please also let me know. i'll create another pr to do this.

Reply:
Ok, I agree with your decision to use fs-extra. It's not urgent to switch to fs/promises, but fewer external dependencies = fewer npm security warnings and maybe slightly faster installation.